docs/user-guide/environment-variables.md

```diff
@@ -64,6 +64,13 @@ _Note: Environment variables set in one job cannot be accessed in another job. T
 | SD_API_URL | Link to the Screwdriver API URL |
 | SD_STORE_URL | Link to the Screwdriver Store URL |
 
+## Template
```

Comment:
Should this go under build specific?

Reply:
That's where i put it originally... lol let me move it back